persistence/test/test.go

```diff
@@ -45,10 +46,21 @@ func ResetPGTestDB(t *testing.T, settings *TestDBSettings) (*pg.DB, string) {
 
 	dbSettings, err := pg.ParseURL(settings.DSN)
 	require.NoError(t, err)
+
+	defaultDb := pg.Connect(dbSettings)
+	dbSettings = defaultDb.Options()
+
+	randDbName := fmt.Sprintf("bottle_test_%d", rand.Uint32()) //nolint:gosec
```

Comment:
I wonder if it could be useful to use an (atomic) counter here, so that if a test fails there aren't so many left-over dbs that should have been dropped. Not on CI, but in a local dev env it may get spammy?

Reply:
What happen if there is a collision for DB names? (for example 2 DB having the same name (unfortunately))? 

We will have the same problem as before.

Like Joost said, it could be good to simply have a counter and increment it (atomic).